index.js

```diff
@@ -214,4 +214,5 @@ function build (opts = {}) {
 
 module.exports = build
 module.exports.prettyFactory = prettyFactory
+module.exports.colorizerFactory = colors
```

Comment:
Could you add a test to cover this new public API?

Reply:
Sure - I've expanded the colorizer test to ensure both exports behave the same! If there's something else you had in mind please let me know. 👍